compiler/src/dmd/dsymbolsem.d

```diff
@@ -8308,3 +8310,55 @@ extern(C++) bool isAbstract(ClassDeclaration cd)
 
     return no();
 }
+
+/*****************************************
+* Is Dsymbol a variable that contains pointers?
+*/
+bool hasPointers(Dsymbol d)
```

Comment:
This is used by LDC so it actually needs to remain `extern(C++)` despite what Iain said. When you add this back to the C++ interface, add it to the `dmd` namespace, not the global one.

Reply:
@thewilsonator this is correct. top-level symbol is `extern(D)`.  The exported C++ symbol is in the `dmd` namespace in cxxfrontend.d.